examples/powercli/datastore-usage-email/handler/script.ps1

```diff
@@ -33,22 +33,34 @@ if( ("$alarmName" -match "$($VC_CONFIG.VC_ALARM_NAME)") -and ([bool]($VC_CONFIG.
     } elseif($alarmStatus -eq "red") {
         $subject = "☢️ $($VC_CONFIG.EMAIL_SUBJECT) ☢️ "
         $threshold = "error"
-    }
+    } elseif($alarmStatus -eq "green") {
+        $subject = "$($VC_CONFIG.EMAIL_SUBJECT)"
+        $threshold = "normal"
+	}
 
-    $Body = @"
-        $alarmName $datastoreName has reached $threshold threshold
+    $Body = "$alarmName $datastoreName has reached $threshold threshold.`r`n"
+	
+	if ( $threshold -ne "normal" )
+	{
+		$Body = $Body + "Please log in to your VMware Cloud on AWS environment and ensure that everything is operating as expected.`r`n"
+	}
 
-        Please login to your VMware Cloud on AWS environment and ensure that everything is operating as expected.
-
-        vCenter Server: $vcenter
+	$Body = $Body + @"
+	    vCenter Server: $vcenter
         Datacenter: $datacenter
-        Datastore: $datastoreName
-
+        Datastore: $datastoreName	
 "@
+	if ($VC_CONFIG.SMTP_PASSWORD.length -gt 0 -and $VC_CONFIG.SMTP_USERNAME.length -gt 0)
+	{
+		$password = ConvertTo-SecureString "$($VC_CONFIG.SMTP_PASSWORD)" -AsPlainText -Force
+		$credential = New-Object System.Management.Automation.PSCredential($($VC_CONFIG.SMTP_USERNAME), $password)
+		Send-MailMessage -From $($VC_CONFIG.EMAIL_FROM) -to $($VC_CONFIG.EMAIL_TO) -Subject $Subject -Body $Body -SmtpServer $($VC_CONFIG.SMTP_SERVER) -port $($VC_CONFIG.SMTP_PORT) -UseSsl -Credential $credential -Encoding UTF32
+	}
+	else
+	{
+		Send-MailMessage -From $($VC_CONFIG.EMAIL_FROM) -to $($VC_CONFIG.EMAIL_TO) -Subject $Subject -Body $Body -SmtpServer $($VC_CONFIG.SMTP_SERVER) -port $($VC_CONFIG.SMTP_PORT) -Encoding UTF32
+	}
```

Comment:
nit: 
* Please indent one-level back
* Starting "{" for If statement should come after the ending paren and not the following line. This reduces the need for another line break just for the brace.

Reply:
You're saying it needs to be aligned with the outer ifcheck $alarmName? It's currently aligned with the #Warning Email Body

![image](https://user-images.githubusercontent.com/29929717/76137293-9c7e2280-6000-11ea-8b59-0bcf30b6dc8b.png)